ofrak_core/Makefile

```diff
@@ -12,10 +12,15 @@ develop: ofrak/gui/public
 .PHONY: inspect
 inspect:
 	mypy
+	
+.PHONY: serial
+serial:
+	$(PYTHON) -m pytest -n0 -m "serial" test_ofrak --cov=ofrak --cov-report=term-missing
+	fun-coverage 
```

Comment:
The order of the serial tests does not matter. What matters is that the two tests are not run in parallel, since one test requires the real `xattr` library be installed, while the other requires it be uninstalled. So to guarantee that they are not run in parallel I have set them to run in serial.